lib/wallet/http.js

```diff
@@ -331,6 +332,28 @@ class HTTP extends Server {
       res.json(200, account.toJSON(balance));
     });
 
+    this.get('/wallet/:id/account/id/:accountIndex', async (req, res) => {
+      const valid = Validator.fromRequest(req);
+      let acct = valid.str('accountIndex');
+
+      const num = parseInt(acct);
+
+      if (!Number.isNaN(num)) {
+        acct = num;
+      }
+
+      const account = await req.wallet.getAccount(acct);
```

Comment:
You will have to create a new method in the `wallet/wallet.js` file and name it `getAccountByAccountIndex` as there is already a method named `getAccountIndex`. I also ask you to name all the references of `getAccountIndex` to `getAccountByAccountIndex`.